src/cozmo/objects.py

```diff
@@ -95,6 +95,17 @@ class EvtObjectAppeared(event.Event):
     pose = 'The cozmo.util.Pose defining the position and rotation of the object.'
 
 
+class EvtObjectAvailable(event.Event):
+    '''Triggered when the engine reports that an object is available (i.e. exists)
+
+    This will usually occur at the start of the program in response to the SDK
+    sending RequestAvailableObjects to the engine.
+    '''
+    obj = 'The object that is available'
+    updated = 'A set of field names that have changed'
+    pose = 'The cozmo.util.Pose defining the position and rotation of the object.'
```

Comment:
The strings assigned to obj and updated don't end with periods but pose does. Should the period at the end of the pose string be removed?